build/makelib/helm.mk

```diff
@@ -36,12 +36,15 @@ $(HELM):
 	@rm -fr $(TOOLS_HOST_DIR)/tmp
 	@$(HELM) init -c
 
+# TODO: after helm 3.0 released just pass --set image.tag=foo
```

Comment:
Is there a timeline for helm 3.0. This sounds great to skip the need for any code generation here.

Reply:
@travisn I haven't found any. 

helm lint have this feature since 2.9, while helm package only in 3.0. I tried to bump helm version locally, and chart as it isn't quite compatible with newer helm versions. imagePullSecrets agent and discover needs to be defined for helm template to work.